pkg/volume/persistent_claim/persistent_claim.go

```diff
@@ -80,6 +81,19 @@ func (plugin *persistentClaimPlugin) NewBuilder(spec *volume.Spec, pod *api.Pod,
 		return nil, err
 	}
 
+	// If a GID annotation is provided set the GID attribute.
+	if volumeGid, ok := pv.Annotations["GID"]; ok {
```

Comment:
The provisioning annotation is `volume.alpha.kubernetes.io/storage-class`, but this PR is not "alpha".

It is probably good to standardize on the "volume" part, too.